packages/editor/src/types/config.ts

```diff
@@ -25,7 +25,20 @@ export type CreateComponentData = { componentName: ComponentType; targetId?: str
 export const isCreateComponentData = (data: unknown): data is CreateComponentData =>
   typeof data === 'object' && data !== null && 'componentName' in data;
 
-type Subsection = 'General' | 'Styling' | 'Behaviour' | 'Options' | 'Static Options' | 'Dynamic Options' | 'Columns' | (string & {});
+export type Subsection =
+  | 'General'
+  | 'Styling'
+  | 'Behaviour'
+  | 'Options'
+  | 'Static Options'
+  | 'Dynamic Options'
+  | 'Columns'
+  | 'Formatting'
+  | 'Parameters'
+  | 'Paginator'
+  | 'Content'
+  | 'Layout'
+  | (string & {});
```

Comment:
remove as this all needs to be translated.
if somewhere a string variable is given, maybe this should be of type Subsection